Makefile

```diff
@@ -70,6 +72,9 @@ update-teleport-repo:
 	@if [ -z "$(TELEPORT_TARGET)" ]; then \
 		echo "TELEPORT_TARGET is not set"; exit 1; \
 	fi
+	@if ! $(type gh); then \
```

Comment:
this line gives me an error unless I change it to 
```
@if ! type gh ; then \
```

Reply:
What shell are you using?

Reply:
bash